.github/CODEOWNERS

```diff
@@ -219,3 +219,5 @@
 /src/image-gallery/ @zhoxing-ms
 
 /src/init/ @zhoxing-ms @HuangYT2000
+
+/src/managementgroup/ @kairu-ms
```

Comment:
@git-thomasdolan Just out of curiosity, is this extension the owned by your team or Azure CLI team? 
If your team is responsible for it's development and maintenance, the owner should be a member from your team

Reply:
Since I generated the commands, I can put myself as an owner